mobify/platform-scaffold
# [WEB-1386] Fix app to work without template styles

As part of the effort to make it as simple as possible to customize and rebrand the website, we want the scaffold's "starting place" to be be devoid of as much custom styles as possible, but still be functional. This PR aims to fix any issues that app might have if all custom template styles (and custom component styles) have been removed.

**JIRA**: [WEB-1386](https://mobify.atlassian.net/browse/WEB-1386)

## Changes
- App.scss: Move the crucial functionality of the App being display: block when the site loads
- `.t-product-list__sort-icon`: move this to component styles for a custom select, not template styles (use `Field` and `Select`, no custom class)
- `.t-checkout-payment__credit-card-hints`: switch to use the `CardVerification` component
- Fix logo SVG to use a font color by default so it’s visible on a white background

## How to test-drive this PR
- Run `npm run dev`
- Preview [Merlin's Potions](https://preview.mobify.com/?url=https%3A%2F%2Fwww.merlinspotions.com%2F&site_folder=https%3A%2F%2Flocalhost%3A8443%2Floader.js&disabled=0&domain=&scope=1)
- Open two SCSS files: `app/styles/_template.scss` and `app/styles/themes/_pw-components.scss`
    - In `_template.scss` comment out all of the imports
    - In `pw-components.scss` comment out just the local theme files (bottom half)
- Reload the previewed app
- Verify that the app's layout and rough styles are still reasonably functional (doesn't have to be perfectly polished, but shouldn't appear broken either)
